plugin.video.thinktv/resources/lib/scraper.py

```diff
@@ -16,91 +16,97 @@
 import sys
 
 h = HTMLParser.HTMLParser()
-
 UTF8 = 'utf-8'
 showsPerPage = 24 # number of shows returned per page by PBS
 
 class myAddon(t1mAddon):
 
  def doPBSLogin(self):
-   if self.addon.getSetting('enable_login') != 'true' : return
-   profile = self.addon.getAddonInfo('profile').decode(UTF8)
-   pdir  = xbmc.translatePath(os.path.join(profile))
-   if not os.path.isdir(pdir): os.makedirs(pdir)
-   cjFile = xbmc.translatePath(os.path.join(profile, 'PBSCookies.dat'))
-   cj = cookielib.LWPCookieJar()
-   badCookie = True
-   try: cj.load(cjFile)
-   except: cj=[]
-   for cookie in cj:
-       if cookie.name == 'sessionid':
-          badCookie = cookie.is_expired()
+    if self.addon.getSetting('enable_login') != 'true':
+        return
 
-   if badCookie:
+    profile = self.addon.getAddonInfo('profile').decode(UTF8)
+    pdir  = xbmc.translatePath(os.path.join(profile))
+    if not os.path.isdir(pdir):
+        os.makedirs(pdir)
+    cjFile = xbmc.translatePath(os.path.join(profile, 'PBSCookies.dat'))
```

Comment:
Actually never mind the above, after looking at the code it's a bit retarded to be saving the cookie  jar instead of just saving the sessionid cookie as a setting. I'll rewrite. thanks.